### PR TITLE
Lowecase read allowed by default

### PR DESCRIPTION
**Pull request by rochacbruno**

### CODE REVIEW

File: dynaconf/__init__.py [hunk 1]
```diff
@@ -9,11 +9,14 @@
 from dynaconf.validator import ValidationError  # noqa
 from dynaconf.validator import Validator  # noqa
 
-settings = LazySettings()
+"""This global settings is deprecated from 3.0.0+"""
+settings = LazySettings(warn_dynaconf_global_settings=True)
+
```
Comment (VaultVulp) [inline dynaconf/__init__.py:14]:
```suggestion
def __construct_singleton_dynaconf_instance():
    warnings.warn(
        "Usage of `from dynaconf import settings` is now "
        "DEPRECATED in 3.0.0+. You are encouraged to change it to "
        "your own instance e.g: `settings = Dynaconf(*options)`",
        DeprecationWarning,
    )
    return LazySettings()

settings = __construct_singleton_dynaconf_instance()
```

In this case we would not need to change `LazySettings` class.

Comment (rochacbruno) [reply dynaconf/__init__.py:14]: I like that, using a FActory/Builder instead of class state, thanks will change it.

Comment (VaultVulp) [reply dynaconf/__init__.py:14]: Sorry, my bad. In this case we will always raise a warning. Even if the user created `LazySettings` instance, in an adequate way.

Comment (rochacbruno) [reply dynaconf/__init__.py:14]: @VaultVulp yes the idea is nice but it is not going to be lazy.

**Pull request merged**